src/playbooks/with_build/upgrade_before_vrs.yml

```diff
@@ -38,11 +38,24 @@
         required_available_disk_space: "{{ vstat_allocate_size_gb }}"
         disk_space_path: "{{ images_path }}"
 
+- hosts: vcins
```

Comment:
The hosts should be "vcin_upgrade_sa_node" as to not trigger on inplace upgrade.